src/modules/assessments/components/pages/ClientAssessmentsPage.tsx

```diff
@@ -82,6 +96,7 @@ const ClientAssessmentsPage = () => {
           noData='No assessments'
           recordType='Assessment'
           defaultSortOption={AssessmentSortOption.AssessmentDate}
+          showOptionalColumns
```

Comment:
What's the purpose of this prop? It seems like it would be easier if the optional column selection appeared by default if any of the columns have the `optional` attribute. But maybe I'm missing something. Are there cases where we pass optional column definitions, but we _don't_ want the selector to show up? I don't see any cases like that, but if there are, maybe having a `hideOptionalColumnSelection` prop would make more sense.